src/utils/helpers.rs

```diff
@@ -103,6 +103,14 @@ pub fn is_choco_installed() -> bool {
     check_output(output)
 }
 
+pub fn is_nargo_installed() -> bool {
```

Comment:
let the name of the function be `is_noir_installed`

Reply:
nargo is the name of the binary, and that is what is used in the official documentation.